### PR TITLE
release proposal 2.1.1

### PR DESCRIPTION
**Pull request by ofrobots**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Node.js Agent for Google Cloud Trace ChangeLog
 
+## 2017-07-17, Version 2.1.1 (Beta), @ofrobots
+
+This module is now in Beta.
+
+### Commits
+
+* [[`17555e2071`](https://github.com/Google Inc./trace-agent/commit/17555e2071)] - beta (#524) (Ali Ijaz Sheikh)
```
Comment (kjin) [inline CHANGELOG.md]: `Google Inc.` -> `GoogleCloudPlatform`

Comment (ofrobots) [inline CHANGELOG.md]: interesting changelog-maker bug.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
 # Node.js Agent for Google Cloud Trace ChangeLog
 
+## 2017-07-17, Version 2.1.1 (Beta), @ofrobots
+
+This module is now in Beta.
+
+### Commits
+
+* [[`17555e2071`](https://github.com/GoogleCloudPlatform/trace-agent/commit/17555e2071)] - beta (#524) (Ali Ijaz Sheikh)
```
Comment (kjin) [inline CHANGELOG.md]: Should have caught this earlier too, but `trace-agent` -> `cloud-trace-nodejs`

Comment (ofrobots) [inline CHANGELOG.md]: Yep. All fixed now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+* [[`ca72dd7f44`](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/commit/ca72dd7f44)] - warn when creating a child of a closed span (#520) (Ali Ijaz Sheikh)
+* [[`be0b006b35`](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/commit/be0b006b35)] - make TraceWriter a singleton (#517) (Kelvin Jin) [#517](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/517)
+* [[`eb0a11be23`](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/commit/eb0a11be23)] - increase severity of module order log message (#519) (Ali Ijaz Sheikh)
+* [[`300dc4fc34`](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/commit/300dc4fc34)] - Fix document source link (#514) (Oleg Shalygin) [#514](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/514)
+* [[`8561232a04`](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/commit/8561232a04)] - Fix typos (#513) (Oleg Shalygin) [GoogleCloudPlatform/cloud-trace-nodejs#513](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/513)
```
Comment (kjin) [inline CHANGELOG.md]: Use just the PR #

Comment (kjin) [inline CHANGELOG.md]: Same for line below

**Pull request merged**